app/models/validator_history.rb

```diff
@@ -30,6 +30,20 @@ class ValidatorHistory < ApplicationRecord
   # Use the monkey patch for median
   include PipelineLogic
 
+  after_create :create_commission_history
```

Comment:
Only after_create? Then where do we store information about each time commission changes?